README.md

```diff
@@ -73,6 +73,7 @@
             * [Create an artifact package file](#create-an-artifact-package-file)
             * [Create an artifact package](#create-an-artifact-package)
             * [Update artifact files of a package](#update-artifact-files-of-a-package)
```

Comment:
I wonder whether we should deprecate this endpoint and mention that they should rather use the new endpoint instead? It's a lot easier to use.

Reply:
The new endpoint is indeed a lot easier to use, but I'm not sure that's a good reason that we should deprecate the other endpoint. Since the other endpoint might have other use cases as well (e.g. if you want to update the artifact with a new set of files and so you don't end up making multiple requests using the newer endpoint)

Reply:
To add new artifacts to a package the new endpoint should always be used as it requires exactly one HTTP call per artifact. Doing it the old way requires one request per artifact file and then one additional request to update the package.
But yes, the old way still has use cases e.g. removing or replacing versions. So deprecating it is not a good idea.

I think the documentation should explain when to use the add and when to use the update endpoint. Can you maybe move the add endpoint before the update endpoint and adjust the update example to show how to remove or replace a version?

Reply:
ok that makes sense. Tried adjusting docs for that.